Makefile

```diff
@@ -35,6 +35,10 @@ publish: build
 	@git push --tags origin HEAD:master
 	@npm publish
 
+build-min: $(LIB)
+	@echo "building lib/TextareaAutosize.min.js"
+	@$(BIN)/rollup -c
```

Comment:
would prefer have input / output files given as input for a `rollup` here rather than configured in a config